docs/en/reference/second-level-cache.rst

```diff
@@ -147,21 +147,21 @@ Built-in cached persisters
 
 Cached persisters are responsible to access cache regions.
 
-    +-----------------------+-------------------------------------------------------------------------------------------+
-    | Cache Usage           | Persister                                                                                 |
-    +=======================+===========================================================================================+
-    | READ_ONLY             | Doctrine\\ORM\\Cache\\Persister\\Entity\\ReadOnlyCachedEntityPersister                    |
-    +-----------------------+-------------------------------------------------------------------------------------------+
-    | READ_WRITE            | Doctrine\\ORM\\Cache\\Persister\\Entity\\ReadWriteCachedEntityPersister                   |
-    +-----------------------+-------------------------------------------------------------------------------------------+
-    | NONSTRICT_READ_WRITE  | Doctrine\\ORM\\Cache\\Persister\\Entity\\NonStrictReadWriteCachedEntityPersister          |
-    +-----------------------+-------------------------------------------------------------------------------------------+
-    | READ_ONLY             | Doctrine\\ORM\\Cache\\Persister\\Collection\\ReadOnlyCachedCollectionPersister            |
-    +-----------------------+-------------------------------------------------------------------------------------------+
-    | READ_WRITE            | Doctrine\\ORM\\Cache\\Persister\\Collection\\ReadWriteCachedCollectionPersister           |
-    +-----------------------+-------------------------------------------------------------------------------------------+
-    | NONSTRICT_READ_WRITE  | Doctrine\\ORM\\Cache\\Persister\\Collection\\NonStrictReadWriteCachedCollectionPersister  |
-    +-----------------------+-------------------------------------------------------------------------------------------+
+    +-----------------------+--------------------------------------------------------------------------------------+
+    | Cache Usage           | Persister                                                                            |
+    +=======================+======================================================================================+
+    | READ_ONLY             | Doctrine\ORM\Cache\Persister\Entity\ReadOnlyCachedEntityPersister                    |
```

Comment:
Github's RST rendering does not seem too happy with the removal of the backslashes… maybe we should use double backticks here?

Reply:
Escaping leads to rendering double backslash on the website :(
 https://www.doctrine-project.org/projects/doctrine-orm/en/2.10/reference/second-level-cache.html#built-in-cached-persisters

Reply:
Ok 👍. I will check it in few days

Reply:
Seems like it works as expected